### PR TITLE
summary of components. A little bit more detailed guide for devs

### PR DESCRIPTION
**Pull request by pabloderivia**
Just a little bit more detailed list of all the components we must create. We could open a new Issue with it too, if you consider it neccesary

### CODE REVIEW

File: front-docs/COMPONENTS_SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#Components Summary
```
Comment (delirial) [inline front-docs/COMPONENTS_SUMMARY.md]: This must be # Component Summary not #Component Summary

File: front-docs/COMPONENTS_SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+##Description
```
Comment (delirial) [inline front-docs/COMPONENTS_SUMMARY.md]: Same than before
# Example
#Example

File: front-docs/COMPONENTS_SUMMARY.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Status legend:
+- P (pending task). the component dev has been not started.
+- VD (Visual Done): the basic aspect (undeed animation can wait) of the component has been coded, but the logic of it its not finished.
+- WL (Working in Visual): someone is working in the html+css
+- LD (Logic Done): all the methods has been established
+- LW (Working in Logic): someone is working the logic of the component
+- TD (Totally done): the component is operating
+
+###Components
```
Comment (delirial) [inline front-docs/COMPONENTS_SUMMARY.md]: Ups

**Review by delirial — CHANGES_REQUESTED** [review]
Some typo errors

File: front-docs/COMPONENTS_SUMMARY.md (hunk 3 above)
Comment (pabloderivia) [reply front-docs/COMPONENTS_SUMMARY.md]: Gotcha

File: front-docs/COMPONENTS_SUMMARY.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- WL (Working in Visual): someone is working in the html+css
+- LD (Logic Done): all the methods has been established
+- LW (Working in Logic): someone is working the logic of the component
+- TD (Totally done): the component is operating
+
+### Components
+
+  - (TD) Generate-Header
+    - (P) Location bar
+    - (VD)Nav buttons (free, custom, short)
```
Comment (armandodecanha) [inline front-docs/COMPONENTS_SUMMARY.md]: (VD) Nav buttons (free, custom, short)

**Review by armandodecanha — APPROVED** [review]
There are some errors in the fragmentation of words, but I don't think it is relevant because it is a documentation for and among us. 

It's ok! 🎉

**Review by delirial — APPROVED** [review]
Good JOB!! This docs grow fast !!! Thanks 💯

**Pull request merged**